dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/route/RouteService.java

```diff
@@ -303,17 +306,22 @@ public ResponseEntity<ResponseBodyEmitter> execute(
     log.debug(
         "Sending '{}' '{}' with route '{}' ('{}')",
         httpMethod,
-        upstreamUrl,
+        upstreamUrlWithoutQueryParams,
         route.getName(),
         route.getUid());
 
     ResponseEntity<Flux<DataBuffer>> responseEntityFlux =
-        retrieve(requestHeadersSpec, httpMethod, upstreamUrl, route.getUid(), userDetails);
+        retrieve(
+            requestHeadersSpec,
+            httpMethod,
+            upstreamUrlWithoutQueryParams,
```

Comment:
Is this where we DO want the query params included? Is this where the actual request is sent?